include/libsemigroups/bipart.hpp

```diff
@@ -250,22 +251,22 @@ namespace libsemigroups {
 
     //! No doc
     template <typename BipartitionOrBlocks, typename Scalar>
-    static void
-    validate_args(std::initializer_list<std::vector<Scalar>> const& blocks) {
+    static void throw_if_bad_args(
+        std::initializer_list<std::vector<Scalar>> const& blocks) {
       std::vector<std::vector<Scalar>> arg(blocks);
-      validate_args<BipartitionOrBlocks>(arg);
+      throw_if_bad_args<BipartitionOrBlocks>(arg);
```

Comment:
I'm not sure what makes the arguments bad, but since this is in detail (and because I don't have a good understanding of the `Blocks` object) this is probably okay.

Reply:
The condition is a bit too long to fit into the function name, hence omitted.